### ISSUE TITLE
Region labels in MQTT topic

### ISSUE THREAD

**Issue by mr-onion-2** [blakeblackshear/frigate#85]
https://community.home-assistant.io/t/local-realtime-person-detection-for-rtsp-cameras/103107/708

It would be useful to distinguish between regions in the MQTT topic so that the same camera can trigger multiple different actions.  Thanks :)

Blake wrote:
> Tripwires and similar features will likely be what you want.

**Comment by scstraus** [comment]
I would love to have it so I could make an imagemap telling it where to scan for humans and another telling it where to scan for cars.

**Comment by blakeblackshear** [comment]
https://www.image-map.net/

https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutorials/py_imgproc/py_contours/py_contours_more_functions/py_contours_more_functions.html#point-polygon-test

**Comment by blakeblackshear** [comment]
I started working on an implementation of "zones". I am thinking it will work something like this:
```
zones:
  #################
  # Name of the zone
  ################
  front_steps:
    ####################
    # For each camera, a list of x,y coordinates to define the polygon of the zone.
    # The same zone can exist across multiple cameras if they have overlapping FOVs.
    # An object is determined to be in the zone based on whether or not the bottom center
    # of it's bounding box is within the polygon. The polygon must have at least 3 points.
    ####################
    cameras:
      front_door:
        - 1,2
    ################
    # Zone level object config. This config takes precendence over the global and camera configs.
    ################
    objects:
      track:
        - person
      filters:
        person:
          min_area: 5000
          max_area: 100000
          threshold: 0.5
  driveway:
  yard:
```

**Comment by blakeblackshear** [comment]
Initial version of with this feature available at `blakeblackshear/frigate:zones-bb8d04b`

**Comment by blakeblackshear** [comment]
See updated example config here: https://github.com/blakeblackshear/frigate/pull/166/files
